TINY-7337: Add toolbar_sticky_offset property to update Sticky toolbar position (#6772)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Unreleased

> - Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337

Reviewer: @camiloHimura this doesn't follow the keep a changelog format sorry, it needs to be something like this instead:
```suggestion

### Added
- Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337
```

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> ## Unreleased
> ### Added

Reviewer: ```suggestion
## Unreleased

### Added
```

## Unreleased

> ### 5.9.0
> 

Reviewer: This should not be here sorry @camiloHimura, it should only be under `Unreleased`.
```suggestion
```

## 5.8.0 - 2021-05-06

### Added

> - Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337

Reviewer: Sorry we've let this stall @camiloHimura 😞 Anyways, this changelog was added to the incorrect location originally so could you please merge in develop and move this to the 5.9/Unreleased section?

- [ ] Move changelog entry

Author: Updated

## 5.4.0 - 2020-06-30

### Added

> - Added new `toolbar_sticky_offset` setting to control sticky header offset #TINY-7337

Reviewer: ```suggestion
- Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337
```
Maybe? Someone else might have a better suggestion yet though.

Author: updated

## 5.4.0 - 2020-06-30

### Added

> - Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337

Reviewer: Don't forget to merge develop and move this to the `Unreleased` section.

- [ ] Move to correct changelog location
